src/appkit.cs

```diff
@@ -21624,6 +21624,919 @@ partial interface NSAccessibilityAttributes {
+		NSString VerticalUnitsAttribute { get; }
+
+		[Mac (10, 6)]
+		[Field ("NSAccessibilityHorizontalUnitDescriptionAttribute")]
+		NSString HorizontalUnitDescriptionAttribute { get; }
+
+		[Mac (10, 6)]
+		[Field ("NSAccessibilityVerticalUnitDescriptionAttribute")]
+		NSString VerticalUnitDescriptionAttribute { get; }
+
+		[Mac (10, 6)]
+		[Field ("NSAccessibilityLayoutPointForScreenPointParameterizedAttribute")]
+		NSString LayoutPointForScreenPointParameterizedAttribute { get; }
+
+		[Mac (10, 6)]
+		[Field ("NSAccessibilityLayoutSizeForScreenSizeParameterizedAttribute")]
+		NSString LayoutSizeForScreenSizeParameterizedAttribute { get; }
+
+		[Mac (10, 6)]
+		[Field ("NSAccessibilityScreenPointForLayoutPointParameterizedAttribute")]
+		NSString ScreenPointForLayoutPointParameterizedAttribute { get; }
+
+		[Mac (10, 6)]
+		[Field ("NSAccessibilityScreenSizeForLayoutSizeParameterizedAttribute")]
+		NSString ScreenSizeForLayoutSizeParameterizedAttribute { get; }
+
+		[Mac (10, 6)]
+		[Field ("NSAccessibilityHandlesAttribute")]
+		NSString HandlesAttribute { get; }
+
+		[Field ("NSAccessibilityDisclosingAttribute")]
+		NSString DisclosingAttribute { get; }
+
+		[Field ("NSAccessibilityDisclosedRowsAttribute")]
+		NSString DisclosedRowsAttribute { get; }
+
+		[Field ("NSAccessibilityDisclosedByRowsAttribute")]
+		NSString DisclosedByRowsAttribute { get; }
+
+		[Field ("NSAccessibilityDisclosureLevelAttribute")]
+		NSString DisclosureLevelAttribute { get; }
+
+		[Field ("NSAccessibilityAllowedValuesAttribute")]
+		NSString AllowedValuesAttribute { get; }
+
+		[Field ("NSAccessibilityLabelUIElementsAttribute")]
+		NSString LabelUIElementsAttribute { get; }
+
+		[Field ("NSAccessibilityLabelValueAttribute")]
+		NSString LabelValueAttribute { get; }
+
+		[Mac (10, 1)]
+		[Obsolete]
+		[Field ("NSAccessibilityMatteHoleAttribute")]
+		NSString MatteHoleAttribute { get; }
+
+		[Mac (10, 1)]
+		[Obsolete]
+		[Field ("NSAccessibilityMatteContentUIElementAttribute")]
+		NSString MatteContentUIElementAttribute { get; }
+
+		[Field ("NSAccessibilityMarkerUIElementsAttribute")]
+		NSString MarkerUIElementsAttribute { get; }
+
+		[Field ("NSAccessibilityMarkerValuesAttribute")]
+		NSString MarkerValuesAttribute { get; }
+
+		[Field ("NSAccessibilityMarkerGroupUIElementAttribute")]
+		NSString MarkerGroupUIElementAttribute { get; }
+
+		[Field ("NSAccessibilityUnitsAttribute")]
+		NSString UnitsAttribute { get; }
+
+		[Field ("NSAccessibilityUnitDescriptionAttribute")]
+		NSString UnitDescriptionAttribute { get; }
+
+		[Field ("NSAccessibilityMarkerTypeAttribute")]
+		NSString MarkerTypeAttribute { get; }
+
+		[Field ("NSAccessibilityMarkerTypeDescriptionAttribute")]
+		NSString MarkerTypeDescriptionAttribute { get; }
+
+		[Mac (10, 7)]
+		[Field ("NSAccessibilityIdentifierAttribute")]
+		NSString IdentifierAttribute { get; }
+	}
+
+	[Static]
+	public interface NSAccessibilityFontKeys {
+		[Field ("NSAccessibilityFontNameKey")]
```

Comment:
But the NSDictionary is returned by the (obsoleted) GetAccessibilityAttributeValue method, which returns a generic NSObject, so we can't using the strong dictionary here either I'm afraid.